### ISSUE TITLE
[BE] - Move research docs into new directories

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#46588]
According to the new platform guidance, we need to move all of our research into a folder (inside our product folder) called `research` and rename directories to follow a specific format: YYYY-MM-Research-initiative-name.

Here's a list of what we need to move, directory by directory

In [profile/combine profile and account/Research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Research):

- [x] [Mobile Sub Navigation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Research/Mobile%20Sub-Navigation) > `profile/combine profile and account/Research/2020-05-mobile-sub-navigation` (need to create folder)
- [x] [UAT](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Research/UAT) > 2020-09-UAT-profile-redesign (need to create folder)
- [x] Move [session-notes](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Research/session-notes) into [2020-03-profile-redesign][(https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Research/2020-03-profile-redesign)

----

In [profile/contact-information](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/contact-information):
- [x] Rename `discovery-and-research` directory to `Research`
- [x] Rename `address-management-april-2020` to `2020-04-address-management`
- [x] In the directory [address-validation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/contact-information/address-validation), rename the `user-testing` directory to `2019-11-address-validation-user-testing` and move it into the newly renamed Research folder you updated in the first checklist item

----

In [/identity-personalization/direct-deposit/discovery-research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/direct-deposit/discovery-research):

- [x] Rename this folder to `Research`
- [x] Rename the `usability-research` folder within it `2019-06-direct-deposit-updates-usability`

-----

In [/profile](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile):
- [x] Rename the [mhv-profile-research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/mhv-profile-research) folder to `2022-05-MHV-profile-research`, and then move it into [/profile/Research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Research)

----

In [/profile/preferred-facility/discovery-and-research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/preferred-facility/discovery-and-research):
- [x] Rename this folder to 'research'
- [x] Move the files inside it into a new directory called `2021-05-preferred-facilities

**Comment by tpharrison** [comment]
Hey @andaleliz - The changes are in the [move-research-docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/move-research-docs) branch.  Do you mind looking over them to make sure everything is ok before I merge the changes?  Thanks!

**Comment by andaleliz** [comment]
Hi @tpharrison! Thanks for doing this! I'm curious how long it took, I think other teams might be interested to learn how you did it.

Everything looks good with one exception, which you attempted to clarify with me and I think I was still confused 🙃 sorry about that!

For the first two items (I unchecked them), can you delete the original folders? So Mobile Sub-Navigation should have been renamed, and _then_ moved. Same with the UAT folder. 

Ideally, the folders starting with the year will be the parent folder for the markdown files, and will not be empty.  Using UAT as an example it would look like:
Combine Profile and Account > Research > 2020-09-UAT-profile-redesign > (markdown files and session notes directory).

Let me know if that's still confusing, but hopefully its not.  Thanks again!

**Comment by tpharrison** [comment]
Got it - I just merged the changes into the [master branch](https://github.com/department-of-veterans-affairs/va.gov-team).  

The changes took about 15 minutes.  I don't know how to do it directly in Github, so I made the changes locally and pushed them to Github.  Here are the high-level steps:

1. Clone the project.
2. Make a new branch.
3. Make updates and commit them. 
4. Push the new branch to Github.
5. Review and merge changes.

I will write up a doc with more detailed steps and explanations.
